docs/spec/manifest-v2-1.md

```diff
@@ -55,18 +55,23 @@ Manifest provides the base accessible fields for working with V2 image format
    
 - **`history`** *array*
    
-   history is a list of unstructured historical data for v1 compatibility.
+   history is a list of unstructured historical data for v1 compatibility. It 
+   contains ID of the image layer and ID of the layer's parent layers.
    
    history is a struct consisting of the following fields
    - **`v1Compatibility`** string
    
-      V1Compatibility is the raw V1 compatibility information.  This  will 
+      V1Compatibility is the raw V1 compatibility information. This will 
       contain the JSON object describing the V1 of this image.
       
 - **`schemaVersion`** *int*
 	
    SchemaVersion is the image manifest schema that this image follows.
 	
+>**Note**:`v1Compatibility` field in `history` array describes the docker runtime configuration 
```

Comment:
Remember, the contents of `v1Compatibility` contain this by convention. There is no enforcement or type structure that actually makes this guarantee.